backend/app/database_mysql.py

```
@@ -323,3 +323,31 @@ def get_user_settings() -> UserSettings:
         logger.error(f"Error getting user settings: {e}")
         raise
+
+def search_consultations_by_keyword(keyword: str) -> List[Consultation]:
+    """
+    Search consultations by keyword using raw SQL.
+    WARNING: This is vulnerable to SQL injection for testing purposes.
+    """
+    try:
+        conn = get_connection()
+        with conn:
+            with conn.cursor() as cursor:
```

Ruff (0.8.2)

Use a single `with` statement with multiple contexts instead of nested `with` statements

Combine `with` statements

(SIM117)